desktop-src/direct3dhlsl_current/hlsl-language-access_qualifiers.md

```diff
@@ -0,0 +1,3 @@
+# Access Qualifiers
+
+See [`DirectX Spec`](https://microsoft.github.io/DirectX-Specs/) for more details.
```

Comment:
What's this page for?  We already have: `hlsl-built-ins-raytracing-payload-access-qualifiers.md`.  This just links to the top-level spec repo, so it's very unhelpful.  This page is probably lefover and should be deleted.